runtime/lua/vim/lsp/util.lua

```diff
@@ -297,6 +297,11 @@ local function get_lines(bufnr, rows)
 
   local filename = api.nvim_buf_get_name(bufnr)
 
+  if vim.fn.isdirectory(filename) then
```

Comment:
```suggestion
  if vim.fn.isdirectory(filename) == 1 then
```
`vim.fn.isdirectory` returns NUMBER (1 or 0).

Reply:
oops, late night PR, thanks :)